shared/src/main/scala/org/akkajs/shocon/SHocon.scala

```diff
@@ -22,7 +23,7 @@ package object shocon extends Extractors {
       lazy val unwrapped = elements.map(_.unwrapped)
     }
     case class Object(fields: Map[Key, Value]) extends Value {
-      lazy val unwrapped = fields.mapValues(_.unwrapped)
+      lazy val unwrapped = fields.view.mapValues(_.unwrapped).toSeq
```

Comment:
I added the toSeq make it compile on:
`shocon/facade/shared/src/main/scala/com/typesafe/config/Config.scala:54:52: value toMap is not a member of Any`
This was not compiling for scala 2.12 only. I think `scala-collection-compat` is not covering this case.
I don't no if it's by design and I should add create some implicit or a defect not yet covered.
@andreaferretti can you have a look? Maybe it's worth opening the bug there.

Reply:
the idiomatic way of doing this should be:
```
fields.view.map(_.unwrapped).to(Seq)
```
according to:
https://www.scala-lang.org/blog/2017/02/28/collections-rework.html

Reply:
ah. has been trying to figure out what the difference was between toSet and to(Seq)